production/ksonnet/loki/config.libsonnet

```diff
@@ -132,6 +137,14 @@
       'config.yaml': $.util.manifestYaml($._config.loki),
     }),
 
+  overrides_config:
+    configMap.new('overrides') +
+    configMap.withData({
+      'overrides.yaml': $.util.manifestYaml({
+        overrides: $._config.overrides,
```

Comment:
do we have to worry about collisions here with cortex overrides? should we put loki overrides in something like `$_config.loki.overrides`?